monitoring/ceph-mixin/dashboards_out/cephfsdashboard.json

```diff
@@ -0,0 +1,4624 @@
+                  "uid": "${datasource}"
+               },
+               "expr": "count(ceph_mds_metadata)",
+               "format": "time_series",
+               "interval": "$interval",
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "refId": "A",
+               "step": 60
+            }
+         ],
+         "title": "Active MDS",
+         "transparent": false,
+         "type": "stat"
+      },
+      {
+         "colors": null,
+         "datasource": {
+            "type": "prometheus",
+            "uid": "${datasource}"
+         },
+         "description": "",
+         "fieldConfig": {
+            "defaults": {
+               "decimals": 0,
+               "links": [ ],
+               "mappings": [
+                  {
+                     "id": 0,
+                     "options": {
+                        "match": "null",
+                        "result": {
+                           "text": "N/A"
+                        }
+                     },
+                     "type": "special"
+                  }
+               ],
+               "thresholds": {
+                  "mode": "absolute",
+                  "steps": [
+                     {
+                        "color": "rgba(50, 172, 45, 0.97)",
+                        "value": null
+                     },
+                     {
+                        "color": "rgba(237, 129, 40, 0.89)",
+                        "value": 0.025000000000000001
+                     },
+                     {
+                        "color": "rgba(245, 54, 54, 0.9)",
+                        "value": 0.10000000000000001
+                     }
+                  ]
+               },
+               "unit": "bytes"
+            }
+         },
+         "gridPos": {
+            "h": 4,
+            "w": 3,
+            "x": 3,
+            "y": 1
+         },
+         "id": 4,
+         "interval": "1m",
+         "links": [ ],
+         "maxDataPoints": 100,
+         "options": {
+            "colorMode": "none",
+            "graphMode": "area",
+            "justifyMode": "auto",
+            "orientation": "horizontal",
+            "reduceOptions": {
+               "calcs": [
+                  "lastNotNull"
+               ],
+               "fields": "",
+               "values": false
+            },
+            "textMode": "auto"
+         },
+         "pluginVersion": "9.4.7",
+         "targets": [
+            {
+               "datasource": {
+                  "type": "prometheus",
+                  "uid": "${datasource}"
+               },
+               "expr": "ceph_pool_bytes_used{pool_id=\"$mdatapool\"}",
+               "format": "time_series",
+               "interval": "$interval",
+               "intervalFactor": 1,
+               "legendFormat": "",
+               "refId": "A",
+               "step": 60
+            }
+         ],
+         "title": "MetaData Used",
```

Comment:
```suggestion
         "title": "Metadata Used (Bytes)",
```

Reply:
Why are we setting the data units in the panel title? I personally disagree with this.

Unit should be set from the `Standard options` menu -> `Units`
And grafana will automatically scale the unit based on the magnitud (https://grafana.com/docs/grafana/latest/panels-visualizations/configure-standard-options/#control-unit-scaling)
Which means that if we set here `(Bytes)`  in the panel title, and we have a balue of `1000000` bytes to display grafana will display `1MB` but we will have `(Bytes)` in title.

Reply:
I did that to differentiate two panels with similar name Metadata Used (Bytes), Metadata Used (%). So to be consistent I added the units in both. If we are fine with just keeping it for Metadata Used (%), I have no issues with that as well

Reply:
Makes sense, thanks for clarifying